package.json

```diff
@@ -13,6 +13,9 @@
     "install": [
       "make install"
     ],
+    "dependencies": {
+      "esy-m4": "esy-packages/esy-m4"
```

Comment:
I would like to avoid using any master branch directly. Can't we publish esy-m4 to npm and then add a proper version range?